newIDE/app/src/InstancesEditor/InstancePropertiesEditor/index.js

```diff
@@ -111,9 +102,10 @@ export default class InstancePropertiesEditor extends Component {
 
   _renderInstancesProperties() {
     const { project, layout, instances } = this.props;
-
+    const instance = instances[0];
+    const object = layout.getObject(instance.getObjectName());
```

Comment:
This won't work if the object does not exist for some reason, and for global objects. Prefer do like in LayerRenderer:

https://github.com/4ian/GDevelop/blob/8866719ef055b08b9e7a494ed04aa9bdd5bd530a/newIDE/app/src/InstancesEditor/InstancesRenderer/LayerRenderer.js#L103-L109

(and if object is null, make sure to don't use it). We could potentially put this in a utility function.

Reply:
where would you prefer to have this utility function? In Utils/GetObjectFromInstance.js ?

Reply:
Sounds like a good file name yeah!